spec/models/ability/response_ability_spec.rb

```diff
@@ -0,0 +1,71 @@
+require "spec_helper"
+
+describe Response do
+  let(:user) { create(:user, role_name: "observer") }
+
+  subject(:ability) do
+    Ability.new(user: user, mission: get_mission)
+  end
+
+  let(:response) { create(:response, form: create_form, answer_values: create_answers) }
+  let(:own_response) { create(:response, form: create_form, answer_values: create_answers, user: user) }
```

Comment:
Is there a reason not to just do a `let(:form) { ...` here? Using `create_form` in this way will create two separate forms I think..?